service/src/main/resources/application.properties

```diff
@@ -11,13 +11,13 @@ logging.level.org.springframework.web: INFO
 logging.level.com.ericsson.eiffel.remrem.producer: INFO
 
 # Jasypt configurations
-jasypt.encryptor.password:<Any value which was used at the time of encrypting the original password will be used at runtime to decrypt the same>
+jasypt.encryptor.jasyptKeyFilePath: <The path of jasypt.key file containing the key value which was used at the time of encrypting the original password will be used at runtime to decrypt the same>
```

Comment:
<The location of the key which was used to encrypt the password, used for decryption of the Encrypted passwords in application >

Reply:
done